src/geometry/fmks.cpp

```diff
@@ -46,6 +46,9 @@ void Initialize<FMKSMeshBlock>(ParameterInput *pin, StateDescriptor *geometry) {
   Real alpha = pin->GetOrAddReal("coordinates", "poly_alpha", 14);
   Real x0 = pin->GetReal("parthenon/mesh", "x1min");
   Real smooth = pin->GetOrAddReal("coordinates", "smooth", 0.5);
+  Real hexp_br = pin->GetOrAddReal("coordinates", "hexp_br", 1.e6);
```

Comment:
Hyperexponential coordinates are on by default... I only see this value being a problem for maybe very large cutout simulations that don't expect hyperexponential coordinates. Maybe this default value should be `std::numeric_limits<Real>::max()`?

Reply:
I would prefer `std::numeric_limits<Real>::max()` yeah

Reply:
`numeric_limits::max()` caused problems so I just used `1.e30` with a check that that's smaller than `Real`.